### PR TITLE
Improve a warning message in TestRunner.

### PR DESCRIPTION
**Pull request by xpconanfan**
Also add another type to gitignore.

### CODE REVIEW

File: mobly/test_runner.py [hunk 1]
```diff
@@ -355,8 +355,8 @@ def get_info(objects):
             self.results.add_controller_info(module_config_name,
                                              controller_info)
         else:
-            self.log.warning("No controller info obtained for %s",
-                             module_config_name)
+            self.log.warning(("No optional debug info found for controller %s, "
+                              "carry on."), module_config_name)
```
Comment (adorokhine) [inline mobly/test_runner.py]: Why the extra set of brackets?

File: .gitignore [hunk 2]
```diff
@@ -9,3 +9,4 @@ Gemfile.lock
 _site
 build/
 dist/
+.DS_Store
```
Comment (adorokhine) [inline .gitignore]: Mind putting this up top to keep this generally in alphabetical order?

File: mobly/test_runner.py [hunk 3]
```diff
@@ -355,8 +355,8 @@ def get_info(objects):
             self.results.add_controller_info(module_config_name,
                                              controller_info)
         else:
-            self.log.warning("No controller info obtained for %s",
-                             module_config_name)
+            self.log.warning(("No optional debug info found for controller %s, "
```
Comment (adorokhine) [inline mobly/test_runner.py]: This still isn't that clear. Can you mention that the controller module doesn't implement get_info?

File: mobly/test_runner.py (hunk 1 above)
Comment (xpconanfan) [inline mobly/test_runner.py]: Isn't it needed to make the string break work?

File: .gitignore [hunk 4]
```diff
@@ -1,4 +1,7 @@
-/.cache/
+build/
+.cache/
```
Comment (adorokhine) [inline .gitignore]: I believe periods sort before letters right?

Comment (xpconanfan) [inline .gitignore]: It is not how the original file sorted:

> *.eggs/
> Gemfile.lock
> .jekyll-metadata

And the official github python gitignore doesn't seem to follow any order.
https://github.com/github/gitignore/blob/master/Python.gitignore

**Pull request merged**